packages/mds-cache/index.ts

```diff
@@ -377,7 +377,7 @@ async function readEvents(device_ids: UUID[]): Promise<VehicleEvent[]> {
     .filter(e => !!e)
 }
 
-async function readAllEvents() : Promise<Array<VehicleEvent | null>> {
+async function readAllEvents(): Promise<Array<VehicleEvent | null>> {
```

Comment:
Not sure how this was making it through in other ppl's PRs. It's a lint fix, and I think it has 0 impact on the codebase.

It'll probably want to get checked in eventually...I can pull it out of this PR if desired.